tooling/nargo/src/ops/foreign_calls.rs

```diff
@@ -60,6 +60,10 @@ impl ForeignCall {
             _ => None,
         }
     }
+
+    pub(crate) fn invalid_name(name: &str) -> bool {
+        ForeignCall::lookup(name).is_none()
```

Comment:
Perhaps turning this into an `is_valid_name` would be more idiomatic, for example we have [BlackBoxFun::is_valid_black_box_func_name](https://github.com/noir-lang/noir/blob/d7a3207b6c3c1c487ce29282730b0ba44481410c/acvm-repo/acir/src/circuit/black_box_functions.rs#L249-L250) which is very similar.

Reply:
I changed the logic, this function is not idiomatic anymore. I tried to use a meaningful name.